lib/main.html

```diff
@@ -132,8 +132,8 @@
   </script>
 
   <script src="../node_modules/@rokt33r/js-sequence-diagrams/dist/sequence-diagram-min.js"></script>
-  <script src="../node_modules/react/dist/react.min.js"></script>
-  <script src="../node_modules/react-dom/dist/react-dom.min.js"></script>
+  <script src="../node_modules/react/umd/react.development.js"></script>
```

Comment:
I assume we need to use another html file for production env because I don't want to use development script in the compiled app.

So what I want to ask are:
1. Rename this file to `lib/main.development.html`
2. Create `lib/main.production.html` by copying from `lib/main.development.html` and update the url so we can use production script.
3. Update declaration statements for `const url` in `lib/main-window.js` so electron can choose the html files by environment.

Reply:
Good catch. I've added this.
I've also updated the link regex or was that not needed?
What do you think should we create an issue to bundle the electron scripts and we don't have to manually import each script? Minifiying could be done by Webpack production build too.

Reply:
> I've also updated the link regex or was that not needed?

I cannot understand could you point where the regex is in?

> What do you think should we create an issue to bundle the electron scripts and we don't have to manually import each script? Minifiying could be done by Webpack production build too.

I absolutely want to do. But let's do things one by one. It should be done in another pr.

Reply:
OK, I'll create an issue for the Webpack update. Then we can also discuss the details there.